src/main/kotlin/com/vaadin/gradle/VaadinCleanTask.kt

```diff
@@ -45,7 +45,8 @@ open class VaadinCleanTask : DefaultTask() {
         project.delete("${project.projectDir}/node_modules",
                 "${project.projectDir}/package.json",
                 "${project.projectDir}/package-lock.json",
-                "${project.projectDir}/webpack.config.js",
+                // don't delete webpack.config.js: https://github.com/vaadin/vaadin-gradle-plugin/issues/43
```

Comment:
I would expect to not need to keep the commented out code in here.

Reply:
It's merely for the sake of warning. I'd advise to keep the line referencing https://github.com/vaadin/vaadin-gradle-plugin/issues/43, but we can delete the commented-out follow-up line if you'd like.

Reply:
Perhaps add a Doc for the whole clean method that mentions that it shouldn't be removed and remove the inline commented lines. 

Also we when this supports 14.2 we should also see to remove `package-lock.yaml` and `pnpm-file.js` should I make a ticket about that?

Reply:
Nah, it's such a tiny change. I'll update this PR, just a sec :)